### PR TITLE
PR for Level 1

### PR DESCRIPTION
**Pull request by shetty-aks**

### CODE REVIEW

File: acme-co/src/components/button/UIButton.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const Button = styled.button<ButtonProps>`
+  background: ${(props) => props.backgroundColor};
+  height: ${(props) => props.height};
+  width: ${(props) => props.width};
+  color: ${(props) => props.color};
+  padding: ${(props) => props.padding};
+  border-radius: 8px;
+`;
+
+const UIButton = ({
```
Comment (alex-kovoy) [inline acme-co/src/components/button/UIButton.tsx]: I would name it `<Button>` and `<StyledButton>`

File: acme-co/src/components/sections/Testimonials.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  align-items: center;
+`;
+
+const Semicolon = styled.span`
+  font-size: 200px;
+`;
+
+const Text = styled.span`
+
+`;
```
Comment (alex-kovoy) [inline acme-co/src/components/sections/Testimonials.tsx:20]: I know that css is not required for the 1st PR but these empty containers with questionable 1 line attributes makes it harder to see the structure. Does `Semicolon` need to be a separate component?

File: acme-co/src/components/layout/SectionWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styled from "styled-components";
+
+type SectionWrapperProps = {
+  background: string;
+  children: ReactNode;
+};
+//botton is just to show the sections till actual styling is added later
+const StyledWrapper = styled.section<SectionWrapperProps>`
+  height:440px;
```
Comment (alex-kovoy) [inline acme-co/src/components/layout/SectionWrapper.tsx]: i suggest to run prettier on every file to format the code

**Review by alex-kovoy — APPROVED** [review]

File: acme-co/src/components/sections/Testimonials.tsx (hunk 2 above)
Comment (shetty-aks) [reply acme-co/src/components/sections/Testimonials.tsx:20]: Hi @alex-kovoy, Thank you for the feedback! So as the layout changes in mobile and desktop view. The position of the semicolon also changes. This is why I chose to make it a separate component. There are a couple of other ways to approach this as well though!

File: acme-co/src/components/layout/SectionWrapper.tsx (hunk 3 above)
Comment (shetty-aks) [reply acme-co/src/components/layout/SectionWrapper.tsx]: Definitely, that helps!

File: acme-co/src/components/button/UIButton.tsx (hunk 1 above)
Comment (shetty-aks) [reply acme-co/src/components/button/UIButton.tsx]: Done, Thanks!

**Pull request merged**